packages/core/src/components/ui/Dialog/Dialog.Wrapper.js

```diff
@@ -27,18 +28,23 @@ const Dialog = forwardRef(function Dialog(props, ref) {
   } = props
 
   useEffect(() => {
-    const handleKeydown = e => {
+    const handleKeydown = (e) => {
```

Comment:
```suggestion
    // eslint-disable-next-line prefer-arrow-callback
    function handleKeydown (e) {
```
`function` should be used here because otherwise `removeEventListener` will not be able to remove listener correctly due to absence of reference